specs/latest/2.0/index.html

```diff
@@ -650,10 +650,6 @@ <h2 class="no-toc">Table of contents</h2>
   const GLenum ANY_SAMPLES_PASSED_CONSERVATIVE               = 0x8D6A;
   const GLenum SAMPLER_BINDING                               = 0x8919;
   const GLenum RGB10_A2UI                                    = 0x906F;
-  const GLenum TEXTURE_SWIZZLE_R                             = 0x8E42;
-  const GLenum TEXTURE_SWIZZLE_G                             = 0x8E43;
-  const GLenum TEXTURE_SWIZZLE_B                             = 0x8E44;
-  const GLenum TEXTURE_SWIZZLE_A                             = 0x8E45;
   const GLenum GREEN                                         = 0x1904;
   const GLenum BLUE                                          = 0x1905;
```

Comment:
GREEN and BLUE enums are only used for texture swizzle and so they could also be removed from here.